solaris/Platform.c

```diff
@@ -255,7 +259,7 @@ static int Platform_buildenv(void* accum, struct ps_prochandle* Phandle, uintptr
       return 1;
    }
    strlcpy( accump->env + accump->size, str, (accump->capacity - accump->size));
-   strncpy( accump->env + accump->size + thissz + 1, "\n", 1);
+   strncpy( accump->env + accump->size + thissz + 1, "\n", 2);
```

Comment:
Is the ` + 1` in this line correct? Because the `strlcpy` only should copy `thissz` bytes into the buffer …

Reply:
It worked&#8482;, it's also `strncpy`.
Sadly valgrind and address sanitizer were not available.